cli/azd/cmd/container.go

```diff
@@ -545,8 +545,8 @@ func registerCommonDependencies(container *ioc.NestedContainer) {
 			return nil, fmt.Errorf("getting platform config: %w", err)
 		}
 
-		if !ok {
-			return nil, errors.New("platform config not found")
+		if !ok || platformConfig.Type == "" {
```

Comment:
If we do prefer this same approach, we should avoid return `type == ""` to mean `ErrPlatformConfigNotFound`

Reply:
See above, an empty `platform.type` is not valid in this use case and in this use case we want it to behave the same as if it was not defined.